app/views/users/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h2>Manage Guiders</h2>
+<% User.guiders.each do |guider| %>
```

Comment:
Nitpick: Can we assign the guiders to an ivar in the controller instead? Just so the controller is explicit about what we reach into with the views.